testing/btest/language/local-blank.zeek

```diff
@@ -0,0 +1,20 @@
+# @TEST-DOC: Do not allow to use the blank identifier for locals.
+
+# @TEST-EXEC-FAIL: zeek -b %INPUT
+# @TEST-EXEC: TEST_DIFF_CANONIFIER=$SCRIPTS/diff-remove-abspath btest-diff .stderr
+event zeek_init()
+	{
+	local _ = "1";
+	}
+
+#@TEST-START-NEXT
+event zeek_init()
+	{
+	local _:string = "1";
+	}
+
+#@TEST-START-NEXT
+event zeek_init()
+	{
+	local _:count= "1";
+	}
```

Comment:
One can currently declare `_` constants (which I think doesn't hurt). Could you add a test for that as well?

```zeek
function foo(): count
	{
	print "foo!";
	return 10;
	}

const _ = foo();
```

Such constants probably shouldn't show up in zeekygen output.

Reply:
I've added a test for using `const`. No zeekygen extras, however.

As long as there's no case for `_` suddenly showing in the docs (you couldn't reference and there's no real reason to use one) I'd postpone.